sc-memory/cpp/sc_template.hpp

```diff
@@ -317,12 +316,13 @@ class ScTemplate final
 
 protected:
   // Begin: calls by memory context
-  bool Generate(ScMemoryContext & ctx, ScTemplateGenResult & result, ScTemplateGenParams const & params, ScTemplateResultCode * errorCode = nullptr) const;
+  bool Generate(ScMemoryContext & ctx, ScTemplateGenResult & result, ScTemplateParams const & params, ScTemplateResultCode * errorCode = nullptr) const;
   bool Search(ScMemoryContext & ctx, ScTemplateSearchResult & result) const;
   bool SearchInStruct(ScMemoryContext & ctx, ScAddr const & scStruct, ScTemplateSearchResult & result) const;
 
   // Builds template based on template in sc-memory
   bool FromScTemplate(ScMemoryContext & ctx, ScAddr const & scTemplateAddr);
+  bool FromScTemplate(ScMemoryContext & ctx, ScAddr const & scTemplateAddr, const ScTemplateParams & params);
```

Comment:
```cpp
bool FromScTemplate(ScMemoryContext & ctx, ScAddr const & scTemplateAddr, ScTemplateParams const & params = ScTemplateParams());
```
One function instead of two:

```cpp
bool FromScTemplate(ScMemoryContext & ctx, ScAddr const & scTemplateAddr);
bool FromScTemplate(ScMemoryContext & ctx, ScAddr const & scTemplateAddr, const ScTemplateParams & params);
```

Reply:
done